util/hack/hack_test.go

```diff
@@ -36,6 +36,8 @@ func TestString(t *testing.T) {
 	if a != "aello world" {
 		t.Fatal(a)
```

Comment:
`t.Fatal(a)` -> `t.Fatalf("a:%v, b:%v", a, b)` to solve the ineffassign warning ?